core/templates/components/skill-selector/merge-skill-modal.component.html

```diff
@@ -5,51 +5,72 @@ <h3>
   <i class="fa fa-times modal-close-button" (click)="cancel()"></i>
 </div>
 <div class="modal-body">
-  <oppia-skill-selector [sortedSkillSummaries]="skillSummaries"
-                        [selectedSkillId]="selectedSkillId"
-                        [countOfSkillsToPrioritize]="countOfSkillsToPrioritize"
-                        [categorizedSkills]="categorizedSkills"
-                        [untriagedSkillSummaries]="untriagedSkillSummaries"
-                        [allowSkillsFromOtherTopics]="allowSkillsFromOtherTopics"
-                        (selectedSkillIdChange)="setSelectedSkillId($event)">
+  <oppia-skill-selector [sortedSkillSummaries]="skillSummaries" [selectedSkillId]="selectedSkillId"
+    [countOfSkillsToPrioritize]="countOfSkillsToPrioritize" [categorizedSkills]="categorizedSkills"
+    [untriagedSkillSummaries]="untriagedSkillSummaries" [allowSkillsFromOtherTopics]="allowSkillsFromOtherTopics"
+    (selectedSkillIdChange)="setSelectedSkillId($event)">
   </oppia-skill-selector>
 </div>
 <div class="modal-footer">
-  <button class="btn btn-success e2e-test-confirm-skill-selection-button"
-          (click)="confirm()"
+  <div class="notice">
+    <span class="notice-warning">Warning:</span>
+    <span class="notice-text">Skills of same topic cannot be Merged!</span>
+  </div>
```

Comment:
Should we only show this message when users try to select skills in the same topic?

Reply:
Yes, we can do that, and I also have another suggestion: we can hide skills which are of same topic..

What are your thoughts on this?

Reply:
Hm, since this is surfaced to a creator, I would recommend that we show a warning / error rather than hiding things. In the creator views we generally follow this principle -- allow the user to make changes they see fit but show relevant errors and prevent save/publish if applicable e.g. validation in the exploration editor page.